### PR TITLE
Add option to build kernels and related tools during chroot and bwrap builds

### PR DESCRIPTION
**Pull request by eduardosm**
The main motivations to have this are:
* Detect checksum changes with chroot/bwrap builds
* Avoid removing checksums from `SHA256SUMS.pkgs` when unsing `--update-checksums` (https://github.com/fosslinux/live-bootstrap/issues/213)
* Find crashes that would otherwise be missed in `builder-hex0` (https://github.com/fosslinux/live-bootstrap/issues/281)

Additionally, this PR also:
* Fixes (or workarounds) https://github.com/fosslinux/live-bootstrap/issues/281.
* Adds checksums for kexec-fiwix and lwext4
* Regenerates SHA256SUMS.pkgs so it matches the file produced with `--update-checksums` (no checkum is change, simply they are sorted)

### CODE REVIEW

**Comment by eduardosm** [comment]
Strange failure, but it [passed in my repo](https://github.com/eduardosm/live-bootstrap/actions/runs/4708714370). I'll retry.

```
2023-04-15T15:17:38.8398001Z i386-unknown-linux-musl-gcc -std=gnu11 -L/usr/lib/i386-unknown-linux-musl  -o Modules/_freeze_importlib Modules/_freeze_importlib.o Modules/getbuildinfo.o Parser/acceler.o Parser/grammar1.o Parser/listnode.o Parser/node.o Parser/parser.o Parser/bitset.o Parser/metagrammar.o Parser/firstsets.o Parser/grammar.o Parser/pgen.o Parser/myreadline.o Parser/parsetok.o Parser/tokenizer.o Objects/abstract.o Objects/accu.o Objects/boolobject.o Objects/bytes_methods.o Objects/bytearrayobject.o Objects/bytesobject.o Objects/cellobject.o Objects/classobject.o Objects/codeobject.o Objects/complexobject.o Objects/descrobject.o Objects/enumobject.o Objects/exceptions.o Objects/genobject.o Objects/fileobject.o Objects/floatobject.o Objects/frameobject.o Objects/funcobject.o Objects/iterobject.o Objects/listobject.o Objects/longobject.o Objects/dictobject.o Objects/memoryobject.o Objects/methodobject.o Objects/moduleobject.o Objects/namespaceobject.o Objects/object.o Objects/obmalloc.o Objects/capsule.o Objects/rangeobject.o Objects/setobject.o Objects/sliceobject.o Objects/structseq.o Objects/tupleobject.o Objects/typeobject.o Objects/unicodeobject.o Objects/unicodectype.o Objects/weakrefobject.o Python/_warnings.o Python/Python-ast.o Python/asdl.o Python/ast.o Python/bltinmodule.o Python/ceval.o Python/compile.o Python/codecs.o Python/dynamic_annotations.o Python/errors.o Python/frozenmain.o Python/future.o Python/getargs.o Python/getcompiler.o Python/getcopyright.o Python/getplatform.o Python/getversion.o Python/graminit.o Python/import.o Python/importdl.o Python/marshal.o Python/modsupport.o Python/mystrtoul.o Python/mysnprintf.o Python/peephole.o Python/pyarena.o Python/pyctype.o Python/pyfpe.o Python/pyhash.o Python/pymath.o Python/pystate.o Python/pythonrun.o Python/pytime.o Python/random.o Python/structmember.o Python/symtable.o Python/sysmodule.o Python/traceback.o Python/getopt.o Python/pystrcmp.o Python/pystrtod.o Python/dtoa.o Python/formatter_unicode.o Python/fileutils.o Python/dynload_shlib.o   Python/thread.o Modules/config.o Modules/getpath.o Modules/main.o Modules/gcmodule.o  Modules/_threadmodule.o  Modules/signalmodule.o  Modules/posixmodule.o  Modules/errnomodule.o  Modules/pwdmodule.o  Modules/_sre.o  Modules/_codecsmodule.o  Modules/_weakref.o  Modules/_functoolsmodule.o  Modules/_operator.o  Modules/_collectionsmodule.o  Modules/itertoolsmodule.o  Modules/atexitmodule.o  Modules/_stat.o  Modules/_localemodule.o  Modules/_iomodule.o Modules/iobase.o Modules/fileio.o Modules/bytesio.o Modules/bufferedio.o Modules/textio.o Modules/stringio.o  Modules/zipimport.o  Modules/faulthandler.o  Modules/_tracemalloc.o Modules/hashtable.o  Modules/symtablemodule.o  Modules/xxsubtype.o -ldl    -lm  
2023-04-15T15:17:39.0326724Z /usr/bin/ld: Python/getversion.o: in function `Py_GetVersion':
2023-04-15T15:17:39.0327551Z /usr/src/python-3.4.10/build/Python-3.4.10/Python/getversion.c:12: undefined reference to `Py_GetBuildInfo'
2023-04-15T15:17:39.0328234Z /usr/bin/ld: Python/sysmodule.o: in function `_PySys_Init':
2023-04-15T15:17:39.0328968Z /usr/src/python-3.4.10/build/Python-3.4.10/Python/sysmodule.c:1694: undefined reference to `_Py_hgversion'
2023-04-15T15:17:39.0330227Z /usr/bin/ld: /usr/src/python-3.4.10/build/Python-3.4.10/Python/sysmodule.c:1694: undefined reference to `_Py_hgidentifier'
2023-04-15T15:17:39.0330726Z make[1]: Leaving directory '/usr/src/python-3.4.10/build/Python-3.4.10'
2023-04-15T15:17:39.0331199Z collect2: error: ld returned 1 exit status
2023-04-15T15:17:39.0331704Z make[1]: *** [Makefile:697: Modules/_freeze_importlib] Error 1
2023-04-15T15:17:39.0332096Z make: *** [Makefile:700: Python/importlib.h] Error 2
```

**Comment by fosslinux** [comment]
That looks suspiciously like a transient parallelism bug. If it comes up again, we can disable parallelism for py 3.4.

**Review by fosslinux — APPROVED** [review]

File: sysa/run2.sh [hunk 1]
```diff
@@ -86,13 +86,15 @@ grep --no-filename '^build' "${SOURCES}"/run*.sh | grep -v musl-1.2.3 | sed "s/b
     rm -rf "${SOURCES:?}/${p:?}"
 done
 
-if [ "${CHROOT}" = False ]; then
+if [ "${CHROOT}" = False ] || [ "${BUILD_KERNELS}" = True ]; then
```
Comment (stikonas) [inline sysa/run2.sh:89]: Maybe force building kernels if update-checksums is specified?

Comment (eduardosm) [reply sysa/run2.sh:89]: Good idea. Done in the last commit.

**Review by stikonas — APPROVED** [review]

**Comment by stikonas** [comment]
> That looks suspiciously like a transient parallelism bug. If it comes up again, we can disable parallelism for py 3.4.

@fosslinux it failed again with Python 3.4. Let's disable parallel builds for it.

**Pull request merged**